### ISSUE TITLE
Spawning Grounds Image named turf-wars-stage-5_2x.jpg

### ISSUE THREAD

**Issue by Dog2puppy** [misenhower/splatoon2.ink#10]
Spawning Grounds is named as if it is a turf war stage, unlike the salmon run stages, which are named lost-outpost.jpg and marooners-bay.jpg.

**Comment by misenhower** [comment]
It's kind of funny, I got those (higher resolution) images from Nintendo's Splatoon 2 site here: https://splatoon.nintendo.com/game-modes/

That specific image is here: https://splatoon.nintendo.com/assets/img/game-modes/turf-wars-stage-5_2x.jpg

I decided to just leave them all in there as they were originally named in case I needed to pull more from their site in the future. SplatNet provides images for each map, but for some reason it only has low-resolution images for the original set of maps. I swap in these higher-resolution images when you click a map on the splatoon2.ink site.

I don't think lost-outpost.jpg or marooners-bay.jpg are used anymore -- I originally used these images when SplatNet didn't provide Salmon Run map/weapon info. Now that it does, I just pull the images provided by SplatNet.

Anyway, good catch, I might clean up some of the unused images eventually 😄
